tests/DeepCat.tests.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @licence GNU GPL v2+
+ * @author Leszek Manicki <leszek.manicki@wikimedia.de>
+ */
+( function( DeepCat, $, QUnit ) {
```

Comment:
Good question. In theory, yes, but QUnit is an external thing we should not change. It's uppercase in QUnit tests in Wikibase.